tests/sources/fixtures/mongodb/fixture.py

```diff
@@ -54,7 +50,3 @@ def remove():
 
     query = {"_id": {"$in": doc_ids}}
     collection.delete_many(query)
-
-
-def teardown():
-    pass
```

Comment:
Is `teardown` ever called? Just confirming that it's safe to remove entirely.

Reply:
`fixture.py` checks if the method is there before calling: 

```
    if hasattr(module, args.action): # <<<<< this
        func = getattr(module, args.action)
        if args.action in ("setup", "load", "teardown", "remove"):
            return await func()
        return func()
```

A lot of our fixtures don't have any of these methods implemented actually (like for SPO, Box, Github, Salesforce) :)